app/services/preserved_object_handler.rb

```diff
@@ -101,22 +101,54 @@ def confirm_version
     results
   end
 
-  def update_version
+  def update_version_after_validation
     results = []
     if invalid?
       results << result_hash(INVALID_ARGUMENTS, errors.full_messages)
     else
-      Rails.logger.debug "update_version #{druid} called"
+      Rails.logger.debug "update_version_after_validation #{druid} called"
```

Comment:
Do we need to know when the method was called in the logs?

Reply:
yes.  It's a "debug" level message, so it will only be written if logging level is set to "debug"